### PR TITLE
Implements covers with location themed images

### PR DESCRIPTION
**Pull request by tonilopezmr**

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -20,7 +20,19 @@
...
+    background-size: cover;   
+}
+
+.overlay {
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+    background-color: rgba(119,193,89,0.5);
```
Comment (alxsimo) [inline css/style.css]: 0.5 is not too clear? can't checkout the branch right now 😢

**Comment by alxsimo** [comment]
👍

File: css/style.css (hunk 1 above)
Comment (tonilopezmr) [reply css/style.css]: It's the same that @SADGevents/valencia, but if you want, when you check the alpha, change it. 

alpha 0.5: 
image

alpha 0.8:

image

Comment (alxsimo) [reply css/style.css]: alpha 0.8 IMHO

**Pull request merged**